src/main/java/com/trueffect/sql/crud/PersonCrud.java

```diff
@@ -1,187 +1,177 @@
 package com.trueffect.sql.crud;
 
-import com.trueffect.response.CorrectResponse;
-import com.trueffect.response.ErrorResponse;
-import com.trueffect.response.MapperResponse;
-import com.trueffect.conection.db.PostgresSQLConnection;
-import com.trueffect.messages.Message;
 import com.trueffect.model.Person;
-
+import com.trueffect.response.ErrorResponse;
 import com.trueffect.tools.CodeStatus;
+
+import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.List;
-import javax.ws.rs.core.Response;
 
 /*
  * @author santiago.mamani
  */
 public class PersonCrud {
 
-    public static Person insertrenterUser(int idJob, Person renterUser) throws Exception {
-      Person renterUserInserted=null;
-        
-            PostgresSQLConnection.connectionDB();
-            try {
-                String typeIdentifier = renterUser.getTypeIdentifier();
-                String identifier = renterUser.getIdentifier();
-                String lastName = renterUser.getLastName();
-                String firstName = renterUser.getFirstName();
-                String genre = renterUser.getGenre();
-                String birthay = renterUser.getBirthday();
-                Statement consulta = (Statement) PostgresSQLConnection.connection.createStatement();
-                String sql=
-                "INSERT INTO person(\n" +
-                " type_identifier, identifier, last_name, first_name, genre, \n" +
-                " birthday, date_create, user_create, date_modifier, user_modifier, \n" +
-                " status)\n" +
-                " VALUES ('"+typeIdentifier+"','"+identifier+"','"+lastName+"', '"
-                        +firstName+"','"+ genre+"','"+birthay+"','2017-12-08','"+idJob+"',null,null,'Active')";
-      
-                consulta.execute(sql);
-                String msg = "The user " + firstName + " " + lastName + " has been successfully inserted";
-                renterUserInserted = getByTypeIdentifier(typeIdentifier, identifier);
-              } catch (Exception e) {
-                throw new ErrorResponse(CodeStatus.INTERNAL_SERVER_ERROR, e.getMessage());
+    public static Person insertrenterUser(Connection connection, int idJob, Person renterUser) throws Exception {
+        Person renterUserInserted = null;
+        Statement query = null;
+        try {
+            String typeIdentifier = renterUser.getTypeIdentifier().trim();
+            String identifier = renterUser.getIdentifier().trim();
```

Comment:
You have this method  Person(int id, String typeIdentifier, String identifier, String lastName, String firstName, String genre, String birthday)  that to the trim of all values .This code be unnecesary